### ISSUE TITLE
Mention `core.excludesFile` in `git clean`'s documentation

### ISSUE THREAD

**Issue by dscho** [gitgitgadget/git#200]
"man git-clean" *explicitly* mentions two locations related to cleaning:

```
  -x
        Don't use the standard ignore rules read from .gitignore
        (per directory) and $GIT_DIR/info/exclude, ...
```

without additionally *explicitly* mentioning core.excludesFile. if the
man page simply said something like, "using the standard ignore
processing" and left it at that, it would be fine, but to list two of
the locations without the third is potentially confusing.

See https://public-inbox.org/git/nycvar.QRO.7.76.6.1902241515120.45@tvgsbejvaqbjf.bet/

**Comment by Denton-L** [comment]
This was closed by b582c1681e (Merge branch 'dl/ignore-docs', 2019-04-10).

**Comment by dscho** [comment]
@Denton-L excellent, thanks!
